helpers/ruby

```diff
@@ -0,0 +1,310 @@
+    if [ -n "$ruby_build" ]; then
+        ynh_print_info --message="\`rbenv install' command already available in \`$ruby_build'."
+        pushd "${ruby_build%/*/*}"
+            if git remote -v 2>/dev/null | grep "https://github.com/rbenv/ruby-build.git"; then
+                ynh_print_info --message="Trying to update rbenv with git..."
+                git pull -q origin master
+            fi
+        popd
+    else
+        ynh_print_info --message="Installing ruby-build with git..."
+        mkdir -p "${rbenv_install_dir}/plugins"
+        git clone -q https://github.com/rbenv/ruby-build.git "${rbenv_install_dir}/plugins/ruby-build"
+    fi
+
+    rbenv_alias="$(command -v "$rbenv_install_dir"/plugins/*/bin/rbenv-alias rbenv-alias | head -1)"
+    if [ -n "$rbenv_alias" ]; then
+        ynh_print_info --message="\`rbenv alias' command already available in \`$rbenv_alias'."
+        pushd "${rbenv_alias%/*/*}"
+            if git remote -v 2>/dev/null | grep "https://github.com/tpope/rbenv-aliases.git"; then
+                ynh_print_info --message="Trying to update rbenv-aliases with git..."
+                git pull -q origin master
+            fi
+        popd
+    else
+        ynh_print_info --message="Installing rbenv-aliases with git..."
+        mkdir -p "${rbenv_install_dir}/plugins"
+        git clone -q https://github.com/tpope/rbenv-aliases.git "${rbenv_install_dir}/plugins/rbenv-aliase"
+    fi
+
+    rbenv_latest="$(command -v "$rbenv_install_dir"/plugins/*/bin/rbenv-latest rbenv-latest | head -1)"
+    if [ -n "$rbenv_latest" ]; then
+        ynh_print_info --message="\`rbenv latest' command already available in \`$rbenv_latest'."
+        pushd "${rbenv_latest%/*/*}"
+            if git remote -v 2>/dev/null | grep "https://github.com/momo-lab/xxenv-latest.git"; then
+                ynh_print_info --message="Trying to update xxenv-latest with git..."
+                git pull -q origin master
+            fi
+        popd
+    else
+        ynh_print_info --message="Installing xxenv-latest with git..."
+        mkdir -p "${rbenv_install_dir}/plugins"
+        git clone -q https://github.com/momo-lab/xxenv-latest.git "${rbenv_install_dir}/plugins/xxenv-latest"
+    fi
+
+    # Enable caching
+    mkdir -p "${rbenv_install_dir}/cache"
+
+    # Create shims directory if needed
+    mkdir -p "${rbenv_install_dir}/shims"
+
+    # Restore /usr/local/bin in PATH
+    PATH=$CLEAR_PATH
+
+    # And replace the old Ruby binary
+    test -x /usr/bin/ruby_rbenv && mv /usr/bin/ruby_rbenv /usr/bin/ruby
+
+    # Install the requested version of Ruby
+    local final_ruby_version=$(rbenv latest --print $ruby_version)
+    if ! [ -n "$final_ruby_version" ]; then
+        final_ruby_version=$ruby_version
+    fi
+    ynh_print_info --message="Installing Ruby-$final_ruby_version"
+    CONFIGURE_OPTS="--disable-install-doc --with-jemalloc" MAKE_OPTS="-j2" rbenv install --skip-existing $final_ruby_version > /dev/null 2>&1
```

Comment:
Apparently this is supposed to be `RUBY_CONFIGURE_OPTS` ?

cf https://github.com/YunoHost-Apps/Experimental_helpers/pull/106 ?

```suggestion
    RUBY_CONFIGURE_OPTS="--disable-install-doc --with-jemalloc" MAKE_OPTS="-j2" rbenv install --skip-existing $final_ruby_version > /dev/null 2>&1
```